docs/config.md

```diff
@@ -66,6 +66,7 @@ module.exports = {
   // configure webpack-dev-server behavior
   devServer: {
     open: process.platform === 'darwin',
+    copy: false,
```

Comment:
`devServer` is used to configure `webpack-dev-server`, I don't think adding a non webpack-dev-server option here is a good idea.